src/services/api/handlers/getPackageActions.ts

```diff
@@ -53,6 +53,7 @@ export const packageActionsForResult = (
       }
     }
   } else if (isStateUser(user)) {
+    actions.push(Action.WITHDRAW_PACKAGE);
```

Comment:
We should also ensure the package is in an active status. Withdrawn is the only currently known "inactive" status (in my brain anyway), so we should double check with Erika and CMS what other statuses may be "inactive" statuses.